### ISSUE TITLE
[1.12] Paxels not dropping Mekanism Ores/Blocks

### ISSUE THREAD

**Issue by OblivionNW** [mekanism/Mekanism#5650]
Making a new issue since I couldn't re-open #5640 

This seems to still be an issue as of commit 7479c88. Tested using an obsidian paxel on a refined obsidian block.

**Comment by pupnewfster** [comment]
Given I am having to fix the CI build anyways (given attempting to release 9.8.2 yesterday didn't work), I went ahead and tested again on the 1.12 branch to make sure this is fixed. Using an obisidian paxel I am able to harvest both osmium ore and refined obsidian blocks (and get them to drop). I will likely push 9.8.2 out in an hour or two and it should work fine from there. Not quite sure why you are unable to mine things.

Note: The only thing I have changed locally from the 1.12 branch is changing the forge version from 2846 to 2847 (as that will be the fix to the CI not working), though I doubt that would change if it works for you or not.
